rpm/prebuild.sh

```diff
@@ -0,0 +1,37 @@
+set_python3() {
+    python3_default=/usr/bin/python3
+    for v in `seq 9 -1 4`
+    do
+      python3=/usr/bin/python3.$v
+      if [[ -e $python ]] && [[ ! -e $python3_default ]]; then
+        echo "Create symlink for Python 3: $python3_default --> $python3"
+        sudo ln -s $python3_default $python3
+        break
+      fi
+    done
+}
+
+if type yum; then
```

Comment:
If you want to use `dnf` on Fedora and `yum` on CentOS, you should check for `dnf` first, because Fedora offers `yum` as alias for `dnf`.

Reply:
Done:

```python
 
-if type yum; then
+if type dnf; then
     VER=$(rpm -E %{rhel})
     # No testing on CentOS 6
     [[ $VER == 6 ]] && exit 0
-    sudo yum install -y python3
+    sudo dnf install -y python3
     set_python3
     if [[ $VER == 7 ]]; then
-        sudo yum install -y python3-devel gcc
+        sudo dnf install -y python3-devel gcc
         python3 -m pip install --user -r test-run/requirements.txt
     fi
-elif type dnf; then
+elif type yum; then
     VER=$(rpm -E %{rhel})
     # No testing on CentOS 6
     [[ $VER == 6 ]] && exit 0
-    sudo dnf install -y python3
+    sudo yum install -y python3
     set_python3
     if [[ $VER == 7 ]]; then
-        sudo dnf install -y python3-devel gcc
+        sudo yum install -y python3-devel gcc
         python3 -m pip install --user -r test-run/requirements.txt
     fi
 else
```

Reply:
script `rpm/prebuild.sh` has been removed